layers/redis/Dockerfile

```diff
@@ -1,13 +1,24 @@
+# syntax = docker/dockerfile:1.4
 ARG PHP_VERSION
-FROM bref/build-php-$PHP_VERSION:1.7.4 AS ext
+FROM bref/build-php-$PHP_VERSION:2 AS ext
+
+RUN <<eot bash -e
+yum install -y php-redis
+cat > /tmp/ext.ini <<INI
+extension=igbinary.so
+extension=msgpack.so
+extension=redis.so
+INI
+eot
 
-RUN pecl install --force redis
-RUN cp `php-config --extension-dir`/redis.so /tmp/redis.so
-RUN echo 'extension=/opt/bref-extra/redis.so' > /tmp/ext.ini
 
 # Build the final image with just the files we need
 FROM scratch
 
 # Copy things we installed to the final image
-COPY --from=ext /tmp/redis.so /opt/bref-extra/redis.so
+COPY --from=ext /usr/lib64/liblzf.so* /opt/lib/
```

Comment:
I think we should do `COPY --link ...`
cf https://youtu.be/mgW61S7izcc?t=886

(took from the video)
![Screenshot 2022-12-11 at 15-33-29 What's New in Docker Build](https://user-images.githubusercontent.com/334432/206909827-e22fce4f-21c7-43db-95d0-0fa90bc103e1.png)

Reply:
Wow that's so interesting! Will probably try to use this in https://github.com/brefphp/[aws-lambda-layers](https://github.com/brefphp/aws-lambda-layers) as well!

Reply:
@mykiwi I've added it https://github.com/brefphp/extra-php-extensions/pull/377/commits/d26f8dd31bd8b57d50c3a884c56da76c209e9803, is it what you had in mind?

Reply:
yes
I just have a little doubt about the `--link` between the `--from=something` & `source dest`, I would probably go for `COPY --link --from=something source dest` but if this does not break :shrug: 
we should probably do this trick on every COPY of the PR (not only for redis)

Reply:
I've tested both, same result. Sounds logic as they just are options so hopefully order has no importance.

:+1: for using it everywhere.